packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -138,13 +140,13 @@ export const VAutocomplete = genericComponent<new <
     function onClickControl () {
       if (
         (props.hideNoData && !items.value.length) ||
-        props.readonly
+        props.readonly || form?.isReadonly
```

Comment:
This condition is always true if form is defined

```suggestion
        props.readonly || form?.isReadonly.value
```